tests/integration/ha_tests/test_ha.py

```diff
@@ -122,6 +132,54 @@ async def test_storage_re_use(ops_test, continuous_writes):
     assert total_expected_writes["number"] == actual_writes
 
 
+@pytest.mark.runner(["self-hosted", "linux", "X64", "jammy", "large"])
+@pytest.mark.group(1)
+@pytest.mark.skip("This is currently unsupported on MongoDB charm.")
```

Comment:
According to our chat in Mattermost I believe the next steps are to research this problem. 

I am okay with keeping this test here as skipped and doing the research separately (but best to remove the DPE-XXX from the title of this PR)

Reply:
Hum.
The ticket was about adding the test, not the support of this feature.
I'd rather keep the ticket id here and create a new one for the research+optional implementation.

Reply:
Yeah maybe this can be added to next pulse to have it working and I can start the investigation on what can be done to support that.